polkadot-host-spec/ag-runtimeentries.tm

```diff
@@ -94,6 +94,17 @@
   The following sections describe the standard based on which the Polkadot
   Host communicates with each runtime entry.
 
+  <section|Managing state changes>
```

Comment:
The appendix isn't really for explaining general aspect of the runtime entries but is for listing specific enteries and their detail. The implementer should get the general rules for implemeting the host from reading the main body of the spec and just comes to the appendix when it needs to call a specific entry. I didn't see this so I added a sub section in the Runtime subsection of the chapter 3. You could move it to the relevant entries in this appendix or merge it with my explanation in the main body of the spec.